app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryModal.tsx

```diff
@@ -0,0 +1,48 @@
+import { Field, Modal, t } from "@bloom-housing/ui-components"
+import React from "react"
+import { useForm } from "react-hook-form"
+import { RailsLotteryBucketsDetails } from "../../api/types/rails/listings/RailsLotteryBucketsDetails"
+import { RailsListing } from "../listings/SharedHelpers"
+import { ListingDetailsLotteryPreferences } from "./ListingDetailsLotteryPreferences"
+
+interface ListingDetailsLotteryModalProps {
+  isOpen: boolean
+  listing: RailsListing
+  lotteryBucketDetails: RailsLotteryBucketsDetails
+  onClose: () => void
+}
+
+export const ListingDetailsLotteryModal = ({
+  isOpen,
+  listing,
+  lotteryBucketDetails,
+  onClose,
+}: ListingDetailsLotteryModalProps) => {
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register } = useForm({ mode: "onChange" })
+
+  return (
+    <Modal onClose={onClose} open={isOpen} title="">
+      <div className="w-96">
+        <header className="pb-4 text-center">
+          <h1 className="">{t("listings.lottery.lotteryResults")}</h1>
```

Comment:
We can likely using the `Heading` component here, so the styles stay consistent https://storybook.bloom.exygy.dev/?path=/story/headers-heading--base-4